node/src/accountant/test_utils.rs

```diff
@@ -321,6 +321,7 @@ impl AccountantBuilder {
     }
 
     pub fn sent_payable_dao(mut self, sent_payable_dao: SentPayableDaoMock) -> Self {
+        // TODO: GH-605: Merge Cleanup
```

Comment:
Only if you want, you can add "Prefer the standard create_or_update_factory! style - as in GH-598"